### PR TITLE
first attempt

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — COMMENTED** [review]
maintainer merge

**Comment by jcommelin** [comment]
maintainer merge

**Review by jcommelin — COMMENTED** [review]
maintainer merge

File: .github/workflows/maintainer_merge_review.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-name: Maintainer merge
+name: Maintainer merge (review)
 
 on:
-  pull_request_review:
+  pull_request_review_comment:
     types: [submitted]
```
Comment (jcommelin) [inline .github/workflows/maintainer_merge_review.yml:5]:
```suggestion
    types: [created, edited]
```

Comment (bryangingechen) [reply .github/workflows/maintainer_merge_review.yml:5]: Will the bot respond if I do this?

maintainer merge

**Comment by bryangingechen** [comment]
maintainer merge

**Pull request merged**